content/singles/ugc-platform.md

```diff
@@ -0,0 +1,119 @@
+---
+title: inBeat UGC Content Creation Platform
+description: >-
+  The UGC content creation platform allows brands & agencies to work with a highly vetted network of Top Social Media creators that generate high quality user generated content. 
+slug: ugc-platform
+layout: ugc-platform
+hero:
+  title: Create UGC that<br/> converts
```

Comment:
You probably don't need the space. If you hide the br on some screen, put the space before.

Also, max-width is often preferred to prevent unexpected cutting on smaller screen but here it would need to be very small.